libcni/api.go

```diff
@@ -223,17 +233,123 @@ func getCachedResult(netName, cniVersion string, rt *RuntimeConf) (types.Result,
 }
 
 // GetNetworkListCachedResult returns the cached Result of the previous
-// previous AddNetworkList() operation for a network list, or an error.
+// AddNetworkList() operation for a network list, or an error.
 func (c *CNIConfig) GetNetworkListCachedResult(list *NetworkConfigList, rt *RuntimeConf) (types.Result, error) {
 	return getCachedResult(list.Name, list.CNIVersion, rt)
 }
 
 // GetNetworkCachedResult returns the cached Result of the previous
-// previous AddNetwork() operation for a network, or an error.
+// AddNetwork() operation for a network, or an error.
 func (c *CNIConfig) GetNetworkCachedResult(net *NetworkConfig, rt *RuntimeConf) (types.Result, error) {
 	return getCachedResult(net.Network.Name, net.Network.CNIVersion, rt)
 }
 
+func stringifyArgs(args [][2]string) string {
```

Comment:
@mccv1r0 let's drop this into a new pkg/util/utils.go along with the ContainerID escaping functions, then use it for skel.go.